en/docs/reference/customize-product/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -1355,14 +1355,26 @@ paths:
         WSDL can be speficied as a single file or a ZIP archive with WSDLs and reference XSDs etc.
         Specify additionalProperties with **at least** API's name, version, context and endpointConfig.
       operationId: importWSDLDefinition
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8" -F file="@wsdl.zip;type=application/zip" -F additionalProperties=@data.json "https://127.0.0.1:9443/api/am/publisher/v1/apis/import-wsdl"'
       requestBody:
         content:
           multipart/form-data:
             schema:
               properties:
                 file:
                   type: string
-                  description: WSDL definition as a file
+                  description: |
+                    WSDL definition as a file or archive
+
+                    **Sample Curl to Upload WSDL File**
```

Comment:
Please use "cURL"

'Sample cURL to upload the WSDL file'

Reply:
Thanks, fixed in 3.1.0 and 3.2.0 PRs with https://github.com/wso2/docs-apim/pull/2556/commits/70839a9d08ffabf15d30b173736acb90c4661c33 and https://github.com/wso2/docs-apim/pull/2552/commits/73bb28df59341834028526ddadba8280fe2a6655